exp-player/addon/randomizers/pref-phys.js

```diff
@@ -206,65 +1450,33 @@ function assignVideos(startType, showStay, whichObjects, NPERTYPE) {
     var allEvents = [];
     var filenames = [];
     var eventNum = 1;
-    for (var nEvents = 0; nEvents < NPERTYPE; nEvents++) {
+    for (var nEvents = 0; nEvents < 6; nEvents++) {
         for (iType = 0; iType < typeOrder.length; iType++) {
             var e = playlistsByType[typeOrder[iType]][nEvents];
 
-            var fname = `sbs_${e.compType}_${e.outcomeL}_${e.outcomeR}_${e.object}_${e.camera}_${e.background}_${e.flip}`;
-            filenames.push(fname);
-            var altName = `sbs_${e.compType}_${e.outcomeR}_${e.outcomeL}_${e.object}_${e.camera}_${e.background}_${e.flip}`;
-            e.fname = fname;
-            e.altName = altName;
+            var fname;
+            var altName;
+            if (e.compType === 'calibration') {
+                fname = `sbs_calibration_${e.flip}`;
+                filenames.push(fname);
+                altName = `sbs_calibration_${e.flip}`;
+                e.fname = fname;
+                e.altName = altName;
+            } else {
+                fname = `sbs_${e.compType}_${e.outcomeL}_${e.outcomeR}_${e.object}_${e.camera}_${e.background}_${e.flip}`;
+                filenames.push(fname);
```

Comment:
Incidentally, this is a truly extraordinary number of videos! I know we saw a few missing videos early on in piloting; amazing to see how many must have come along since.